sh/cmd.go

```diff
@@ -89,6 +89,19 @@ func OutputWith(env map[string]string, cmd string, args ...string) (string, erro
 	return strings.TrimSuffix(buf.String(), "\n"), err
 }
 
+// BothOutput runs the command and returns the text from stdout and stderr.
+func BothOutput(cmd string, args ...string) (string, string, error) {
```

Comment:
`CombinedOutput` would make more sense as the name of this function.
https://golang.org/pkg/os/exec/#Cmd.CombinedOutput